tests/installation/network_check.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright 2025
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Make sure NIC is up and ip address
+#          is allocated after installation
+#
+# - We only need to care about QEMU backend
+# Maintainer: QE Core <qe-core@suse.de>
+
+use strict;
+use warnings;
+use base 'consoletest';
+use testapi;
+use Utils::Backends 'is_qemu';
+
+sub qemu_backend_ifup_check {
+    return unless is_qemu;
+    if (get_var('IFUP_CHECK', '1')) {
+        my $vm_ip_addr = get_var('VM_IP_ADDR', '10.0.2.15');
+        my $vm_ip_route = get_var('VM_IP_ROUTE', '10.0.2.2');
```

Comment:
I recommend to use `testapi::host_ip()` as default value. This function is used by many others including `testapi::autoinst_url()` or `network_utils::setup_static_network()`.